### ISSUE TITLE
Can not connect to MariaDB Database

### ISSUE THREAD

**Issue by Alex-Dall** [rvrn22/nmg#14]
Hello,
I have a problem using this program: I can not connect to my MariaDB database. The database is reachable with other tools, only with this one i can't connect. 
I don't know if i do something wrong or if it is a bug.

I have done the following setttings:
Database Type: MySQL
OLE DB Provider: MSDataShape
Server or file name: IpAddress:Port
Location: 
Use a specific user name and password: My username and password

If I clilck on "Test Connection" it says "Test conncetion succeded" (But I can write anything into the fields. As long as I leave MSDataShape the Test Connection "works")

The connection string generated: 
`Provider=MSDataShape.1;Data Source=####.####.#.##:####;User ID=root;Password="#########"`

But when I click on "Connect" it gives me the following error in the status bar at the bottom:
`Keyword not supported. Parametername: provider`

I would be really happy if you could help me.

Alex

PS: If have tried it with the latest compiled release and I have also built the on my one with the code GitHub.

**Comment by Vendare** [comment]
That seems to be a major oversight the provider is simply broken or not integrated correctly. Tried using the Tool for a PostgreSQL Database. Using the MSDataShape provider results in the same error message.

**Comment by werkwolk** [comment]
yep, same here...unfortunate, this project is basically abandoned

**Comment by werkwolk** [comment]
if you simply upgrade the npgsql reference to 4.06 and fix the connectionstring in NpgsqlMetadataReader it works fine.

good luck!
